config/helpers/bootstrap_test.go

```diff
@@ -0,0 +1,76 @@
+/*
+Copyright 2021 The KCP Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package helpers
+
+import (
+	"context"
+	"embed"
+	"testing"
+	"time"
+
+	crdfake "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset/fake"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/kubernetes/fake"
+)
+
+type mockDynamic struct {
+	client *fake.Clientset
+}
+
+func (m mockDynamic) Resource(gvr schema.GroupVersionResource) dynamic.NamespaceableResourceInterface {
+	return nil
+}
+
+func TestBootstrap(t *testing.T) {
+	crds := []metav1.GroupResource{
+		//	{Group: tenancy.GroupName, Resource: "workspaces"},
```

Comment:
no crds?

Reply:
I have to mock them in the client or the test doesn't go through, I was trying to make my head around this